packages/grafana-data/src/field/standardFieldConfigEditorRegistry.ts

```diff
@@ -24,6 +24,7 @@ export interface StandardEditorProps<TValue = any, TSettings = any, TOptions = a
 export interface StandardEditorsRegistryItem<TValue = any, TSettings = any> extends RegistryItem {
   editor: ComponentType<StandardEditorProps<TValue, TSettings>>;
   settings?: TSettings;
+  defaultValue?: TValue;
```

Comment:
@dprokop -- I am surprised we don't have this already, but can not find it anywhere else 🤷 

The values defined in [options.ts](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/options.ts#L49) get passed to each editor, but defaultValue is expected to be applied already

Reply:
I can find it here on the instanced options,  (FieldConfigPropertyItem inherits this)
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/options.ts#L49

But there is no default on the editor definitions StandardEditorsRegistryItem.

Reply:
Yeah, the default was only implemented on the option config. There was no default on the registry item itself as it's not obvious what the default should be when i.e. RadioButtonGroup option is created 😬

Reply:
See my comment below, I don't think we need this property on `StandardEditorsRegistryItem` as it may be a consequence of the actual option config (i.e. Radio or Slider editors).